### DISCUSSION TITLE
Union of two polygons that in result creates hole

### DISCUSSION THREAD

**Discussion by mindbrave** [alexbol99/flatten-js#190] [Q&A]
First I just want to ask if it's supported, because I have a feeling that it's not. I created polygon like that 
<img width="391" alt="image" src="https://github.com/user-attachments/assets/8d9b0d22-f50d-41fa-9abf-86d34ff287e1">
and then I want to close it from top, which results in a hole inside, but result I get is this: 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/a17cae34-c697-4642-a994-2e8a1b0ff325">

I looked what is inside of that polygon and there is still only one face, so I suppose it's not creating whole faces automatically? Can I fix that in some different way?

Thanks for a great library by the way.

**Comment by alexbol99** [comment]
I need to understand how exactly you “ closed it from top”? If it was done using unify boolean operation, then indeed the result should be a polygon with one hole. 
If it is not, I need a data to investigate the case.

**Reply by mindbrave** [reply]
yes, I used `unify` to close it from top.
Sorry, I must have messed something, when I tried to create example data for you then it worked as expected and created holes. I think my rendering is broken, I am using pixi.js to render polygons.
